appshell/appshell_extensions_platform.h

```diff
@@ -55,15 +62,18 @@ inline void* getMenuParent(CefRefPtr<CefBrowser>browser) {
     }
     return NULL;
 }
+inline int32 InstallCommandLineTools() { return ERR_CL_TOOLS_NOTSUPPORTED; }
```

Comment:
Yes. This is to make sure that when this API is called on platforms other than Mac, we are going to return an error.